renderer.js

```diff
@@ -44,6 +44,23 @@ function bindCanvas(c) {
   // renderer.gammaFactor = 2.2;
   renderer.shadowMap.enabled = true;
   renderer.shadowMap.type = THREE.PCFSoftShadowMap;
+
+  // reload renderer WebGLInfo.reset function
+  // in order to continue chaning frames value
```

Comment:
What is a chaining frames value?

Reply:
Changing renderer.info.frames number.
Is increased each .info.reset() call.
If not called automatically [renderer.info.autoReset = false] it'll result in renderer.info.frames not updating and will cause issues.

Reply:
Where is the chain?

Reply:
> If not called automatically [renderer.info.autoReset = false] it'll result in renderer.info.frames not updating and will cause issues.

What issues?